src/Tasks/Microsoft.NET.Build.Tasks/LockFileExtensions.cs

```diff
@@ -169,10 +208,47 @@ public static LockFileTargetLibrary GetLibrary(this LockFileTarget lockFileTarge
             string assetType)
         {
             return library.RuntimeTargets
-                .FilterPlaceHolderFiles()
+                .FilterPlaceholderFiles()
                 .Cast<LockFileRuntimeTarget>()
                 .Where(t => string.Equals(t.AssetType, assetType, StringComparison.OrdinalIgnoreCase))
                 .GroupBy(t => t.Runtime);
         }
+
+
+        // A package is a TransitiveProjectReference if it is a project, is not directly referenced,
+        // and does not contain a placeholder compile time assembly
```

Comment:
What would cause there to be a project in the assets file that meets the other conditions but has a placeholder file?

Reply:
I couldn't repro it. I talked to @emgarten and it has something to do with assets flags on project refs, but none that I tried caused it. This is just moved from elsewhere.